tests/components/flexmeasures/test_services.py

```diff
@@ -24,3 +34,62 @@ async def test_change_control_type_service(hass: HomeAssistant, setup_fm_integra
             blocking=True,
         )
         mocked_CEM.assert_called_with(control_type=ControlType.NO_SELECTION)
+
+
+async def test_trigger_and_get_schedule(
+    hass: HomeAssistant, setup_fm_integration
+) -> None:
+    """Test that the method activate_control_type is called when calling the service active_control_type."""
```

Comment:
There are two test functions here with different names and the exact same docstring. That means at least one of the docstrings is incorrect. Probably both.